man/manual.md

```diff
@@ -1193,7 +1188,8 @@ Commands:
     cd check-path
 
     # Calculate lines of code and validate the license
-    premiumaddon --check-loc-license some-path/license-file > cppcheck-premium-loc
+    premiumaddon --check-loc-license some-path/license-file \
```

Comment:
The open source manual should NOT say how premiumaddon is executed. Remove the whole "Cppcheck Premium" chapter. Especially the "Licenses" is out of scope.